src/server.h

```diff
@@ -1975,6 +1989,18 @@ struct valkeyServer {
                                                 * delay (start sooner if they all connect). */
     int dual_channel_replication;              /* Config used to determine if the replica should
                                                 * use dual channel replication for full syncs. */
+    _Atomic int replica_bio_disk_save_state;   /* Flag set by the bio thread to indicate that the
+                                                * RDB save to disk has completed, or failed */
+    _Atomic int replica_bio_abort_save;        /* Flag set by main thread, used to signal to replica's
```

Comment:
why not bool?

Reply:
`replica_bio_disk_save_state` can hold many values, but for `replica_bio_abort_save` you are correct